CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file. The format
 
 ## [Unreleased]
 
+## [v2.0.0] - 2025-01-13
```

Comment:
Remove this, this will be generated from the pipeline. Leave it in **Unreleased**